app/com/linkedin/drelephant/schedulers/AzkabanScheduler.java

```diff
@@ -73,6 +75,8 @@ private void loadInfo(String appId, Properties properties) {
 
     workflowDepth = 0; // TODO: Add sub-workflow support
     jobName = properties.getProperty(AZKABAN_JOB_NAME);
+    organization = properties.getProperty("job.billing.org");
```

Comment:
Please take these out in constants section.

Reply:
Also does dr elephant have an abstract scheduler? If yes then org, namespace interpretation and logic can probably go there. Also I dont see a need to override them in the concrete impls.